docs/06_how-to-guides/30_key-value-api/kv_table/20_how-to-create-indexes-kv-table.md

```diff
@@ -54,14 +54,14 @@ class [[eosio::contract]] smrtcontract : public contract {
 
 ## Procedure
 
-### Define a unique index on property account_name using the macro KV_NAMED_INDEX
+### Define a unique index on data member account_name using the macro KV_NAMED_INDEX
 
 1. Use the `KV_NAMED_INDEX` macro with two parameters.
-2. Pass the name of the index as the first parameter. The parameter must be a qualified `eosio::name`. See documentation for the [eosio name restrictions](https://developers.eos.io/welcome/latest/glossary/index#account-name).
-3. Pass the name of the property for which the index is defined as the second parameter.
-4. Call `init()` of the base class in the constructor of `address_table` type and pass the contract name as the first parameter and the primary index defined previously as the second parameter.
+2. The first parameter is the name of the index and it must be a qualified `eosio::name`. See documentation for the [eosio name restrictions](https://developers.eos.io/welcome/latest/glossary/index#account-name).
+3. The second parameter is the name of the data member the index is defined for.
```

Comment:
The second parameter is the name of the data member the index is defined for.

A bit pedantic but =>

The second parameter is the name of the data member the index is defined ON.

Reply:
:) I see the equivalent, no?

Reply:
no ... for is 'clunky'.

Reply:
+ are steps 2 and 3 steps used to define the KV_NAMED_INDEX?

This should be reflected by making 2 and 3 indented below 1.

Reply:
or should 1. even be in a numbered list - isn't this a description of what you are doing?